neuralprophet/forecaster.py

```diff
@@ -701,6 +758,15 @@ def fit(self, df, freq="auto", validation_df=None, progress="bar", minimal=False
                     If you see no impact of the regularization, turn off the early_stopping or reduce the number of epochs to train for."
                 )
 
+        if progress == "plot" and metrics is False:
+            log.warning("Progress plot requires metrics to be enabled. Enabling the default metrics.")
+            metrics = metrics.get_metrics(True)
+
+        if minimal:
+            checkpointing = False
+            self.metrics = False
+            progress = None
```

Comment:
Consider removing the minimal parameter or add a warning in case any other configuration (eg. checkpointing etc) is provided together with minimal.